cli/main.go

```diff
@@ -260,7 +260,7 @@ func main() {
 		handleError(ctx, err, ctype, currentContext, cc, root, start, duration)
 	}
 	metricsClient.Track(
-		metrics.CmdMeta{
```

Comment:
(Sorry for the rename - there's so many types with `command` or `event` in their name since we've got 2x very similar code paths here now, so I've been _trying_ to keep things clear)